### PR TITLE
Add page for matrix-rss-bridge

### PR DESCRIPTION
**Pull request by imbev**
Preview: https://pr1435--matrix-org-previews.netlify.app

### CODE REVIEW

File: gatsby/content/projects/bridges/matrix-rss-bridge.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: project
+title: matrix-rss-bridge
+categories:
+ - bot
+ - bridge
```
Comment (thibaultamartin) [inline gatsby/content/projects/bridges/matrix-rss-bridge.mdx:6]:
```suggestion
```

This seem to be more of a bot than a bridge

File: gatsby/content/projects/bridges/matrix-rss-bridge.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+description: matrix-rss-bridge is a bridge for reading RSS feeds in Matrix rooms.
```
Comment (thibaultamartin) [inline gatsby/content/projects/bridges/matrix-rss-bridge.mdx:7]:
```suggestion
description: matrix-rss-bridge is a bot for reading RSS feeds in Matrix rooms.
```

File: gatsby/content/projects/bridges/matrix-rss-bridge.mdx (hunk 1 above)
Comment (imbev) [reply gatsby/content/projects/bridges/matrix-rss-bridge.mdx:6]: It is both a bot and a bridge, similar to feedbot which is categorized as both.

Comment (thibaultamartin) [reply gatsby/content/projects/bridges/matrix-rss-bridge.mdx:6]: Ah good catch, this is a mistake: at first sight feedbot is not a bridge either. The major difference between a bridge and a bot is that bridges a bi-directional.

I might have missed it, but I don't think this is the case for this project, is it?

Comment (thibaultamartin) [reply gatsby/content/projects/bridges/matrix-rss-bridge.mdx:6]: I suggest to remove the bridge section for now, and to add it in the future if you add features to make it bi-directional. Are you fine with me applying the suggestions and merging the PR?

Comment (imbev) [reply gatsby/content/projects/bridges/matrix-rss-bridge.mdx:6]: Yes

**Pull request merged**